yjit/src/stats.rs

```diff
@@ -724,6 +724,14 @@ fn rb_yjit_gen_stats_dict() -> VALUE {
         }
     }
 
+    macro_rules! hash_aset_flonum {
+        ($hash:ident, $key:expr, $value:expr) => {
+            let key = rust_str_to_sym($key);
+            let value = VALUE::flonum_from_double($value);
+            rb_hash_aset($hash, key, value);
+        }
+    }
```

Comment:
Seems like this could be a function rather than a macro? but it's a good idea to encapsulate this functionality 👍

Reply:
I did it that way because that's what was done for fixnums right above it. 🤷 
That does seem sensible to me though. I'm happy to convert both of them to functions.